src/mon/MonCommands.h

```diff
@@ -645,6 +645,9 @@ COMMAND("osd crush tree "
         "name=shadow,type=CephChoices,strings=--show-shadow,req=false", \
 	"dump crush buckets and items in a tree view",
 	"osd", "r", "cli,rest")
+COMMAND("osd crush ls name=node,type=CephString,goodchars=goodchars=[A-Za-z0-9-_.]",
```

Comment:
nit: for coding style consistency, might want to append an extra **\** at each end of line..